rest-service/api/src/main/java/org/veupathdb/service/multiblast/controller/JobController.java

```diff
@@ -50,7 +50,7 @@ public Response getJobs() {
    * @return Basic info about the newly created job (such as the job id).
    */
   @Override
-  public Response postJobs(IOJsonJobRequest entity) {
+  public Response postJob(IOJsonJobRequest entity) {
```

Comment:
Renamed generated methods because reasons.  (Code generation is long since disabled on this repo)